### ISSUE TITLE
Just a question about Github, will be deleted

### ISSUE THREAD

**Issue by NicolasTurpin** [NicolasTurpin/Touhou-Gensou-Mahjong-4N---ENGLISH-translation#6]
While exploring Github I saw ther's a way to enguage discutions (instead of openning issues around the game)
Do somebody know where it is ? x)

**Comment by NicolasTurpin** [comment]
PS: I saw some people forking the project.
I don't know much about Git (even if I worked with it for 4 years...) so what does this mean ? I guess they are making backups in some sort of way but can I push something so their fork are updated ?

**Comment by remi-sc** [comment]
Most of github discussions happen in issues or pull requests. AFAIK there aren't any better options here.

Forks are essentially copies that someone else can work on independently and when changes are created on their side they can create a pull request here - Essentially asking you to pull their changes into your tree.
You have no option to force someone to accept any updates. It's their copy, and they will decide if/when to update.

But honestly this repository is really horrible for any organic contributions. It's just binary blobs that aren't diff-able.
If you want a proper way to receive contributions and discuss the changes then you would need to switch from packed, encrypted version to unpacked sources with a key+script+readme how to repack everything (I do something like this myself locally)

And distribute binary blobs as attachments to "releases/tags".

**Comment by NicolasTurpin** [comment]
Hey !  I just came back from a short travel, I'm getting back to the translation !

I'm adding few things to my todo list and reading what you said :

for the Fork : Roger that !
for the repo : ... yeah ^^" I'm sorry for the massive mess I'm sure I'll find a way to clean it up but in any case, I only put crypted packages because I don't think D.N.A. software would be very happy to have their files shown publicly. 
It's quite a shame and a pain in the ass but I'll stick with it for now (unless I got the permission from the devs but my JP friend is taking a bit of time to help me write a proper mail ^^")

**Comment by NicolasTurpin** [comment]
I'm feeling a bit hypocrytical as I the key and uncrypt-script was kinda public from the start but let's minimize the damages haha

**Comment by remi-sc** [comment]
It's basically game modding but I am not sure how big of a dinosaurs are people from DNA.
I can only suggest to stress the points that this project requires a legal copy to function and that no advantage is given when playing online.

**Comment by NicolasTurpin** [comment]
Yeah... And I can't imagine somebody using this to cheat, even so thay can't recompile the game without the key x)
I'll ask D.N.A, even without a proper mail I can at least join them on twitter and tell them about the project.

I'm just going to translate the Music names (researching the official english names for some) and push this out.

If it works for them I'll re-work all the git project with the uncrypted files for work-sharing !
